conformance/packages/dns-test/src/container.rs

```diff
@@ -227,6 +227,8 @@ impl Container {
         command
             .args(["exec", &self.inner.id])
             .args(command_and_args);
+        command.stdout(Stdio::null());
```

Comment:
Can we chain these onto the existing `.args()` calls? Seems like it should work?